examples/single_agent.py

```diff
@@ -70,6 +71,7 @@ def main(scenarios, headless, num_episodes, max_episode_steps=None):
 if __name__ == "__main__":
     parser = default_argument_parser("single-agent-example")
     args = parser.parse_args()
+    sys.path.append(".")
```

Comment:
Why was this needed?

Reply:
I cannot run the simulation sometimes without this line, I will remove this once other changes are approved